src/main/java/org/jahia/modules/sam/healthcheck/probes/SupportedStackJDKProbe.java

```diff
@@ -0,0 +1,93 @@
+package org.jahia.modules.sam.healthcheck.probes;
+
+import org.graalvm.polyglot.Context;
+import org.jahia.bin.Jahia;
+import org.jahia.commons.Version;
+import org.jahia.modules.sam.Probe;
+import org.jahia.modules.sam.ProbeSeverity;
+import org.jahia.modules.sam.ProbeStatus;
+import org.jahia.osgi.BundleUtils;
+import org.osgi.framework.Bundle;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+@Component(service = Probe.class, immediate = true)
+public class SupportedStackJDKProbe implements Probe {
+
+    @Override
+    public ProbeStatus getStatus() {
+        Bundle npmModulesEngineBundle = BundleUtils.getBundleBySymbolicName("npm-modules-engine", null);
+        boolean isNpmModulesEngineStarted = npmModulesEngineBundle != null && npmModulesEngineBundle.getState() == Bundle.ACTIVE;
+
+        String vmVendor = System.getProperty("java.vm.vendor");
+        Version jahiaVersion = new Version(Jahia.VERSION);
+        Version jdkVersion = new Version(System.getProperty("java.version", "Unknown"));
+
+        ProbeStatus status =  new ProbeStatus("Jahia version and your JVM version are compatible", ProbeStatus.Health.GREEN);
+
+        if (jahiaVersion.compareTo(new Version("8.2.0.O")) < 0) {
+            if (jdkVersion.compareTo(new Version("1.8")) >= 0 && jdkVersion.compareTo(new Version("11")) <= 0) {
+                status = updateStatus(status,"Jahia version and JVM version are not compatible, use version 8 or version 11", ProbeStatus.Health.RED);
+            }
+            if (!vmVendor.contains("Oracle") && !vmVendor.contains("Eclipse")) {
+                status = updateStatus(status, "Current Jahia version is compatible with Eclipse Adoptium or Oracle jvm vendors", ProbeStatus.Health.YELLOW);
```

Comment:
I believe here the critical part in Adoptium Eclipse Temurin is actually Temurin. Adoptium and Eclipse are two vendors while temurin is the name of the project. If Eclipse (or Adoptium) goes on to create another JDK build, it will not necessarily mean we support it.

Reply:
After searching in all java.vm properties, no one contains Temurin